_dev/src/components/smart-shopping-campaign/smart-shopping-campaign-table-list-row.vue

```diff
@@ -93,12 +96,18 @@ export default {
     campaignCountryName() {
       return this.$options.filters.changeCountriesCodesToNames([this.campaign.targetCountry])[0];
     },
+    totalProducts() {
+      return retrieveProductNumberFromFiltersIds(
+        this.campaign.productFilters, this.$store.state.smartShoppingCampaigns.sscAvailableFilters,
+      );
+    },
     campaignProducts() {
       return this.campaign.productFilters?.length
         // eslint-disable-next-line
-        ? this.$i18n.tc('smartShoppingCampaignCreation.nbProductsSelected',
-          this.campaign.productFilters[0].values.length,
-          [this.campaign.productFilters[0].values.length])
+        ? this.$i18n.t('smartShoppingCampaignCreation.dimensionXFilterSelected',
+          [this.$i18n.t(`smartShoppingCampaignCreation.${this.campaign.productFilters[0].dimension}`)])
+          + ' (' + this.$i18n.tc('smartShoppingCampaignCreation.xProducts', this.totalProducts,
+          [this.totalProducts]) + ')'
```

Comment:
Too complex, it will be better to avoid a whole translated string in which you write the expected format in English (i.e `blabla (xx products)` instead of concatenating strings here